react/features/analytics/handlers/GoogleAnalyticsHandler.js

```diff
@@ -0,0 +1,153 @@
+/* global ga */
+
+import { getJitsiMeetGlobalNS } from '../../base/util';
+
+import AbstractHandler from './AbstractHandler';
+
+const logger = require('jitsi-meet-logger').getLogger(__filename);
+
+/**
+ * Analytics handler for Google Analytics.
+ */
+class GoogleAnalyticsHandler extends AbstractHandler {
+
+    /**
+     * Creates new instance of the GA analytics handler.
+     *
+     * @param {Object} options -
+     * @param {string} options.googleAnalyticsTrackingId - The GA track id
+     * required by the GA API.
+     */
+    constructor(options) {
+        super();
+
+        this._userProperties = {};
+
+        if (!options.googleAnalyticsTrackingId) {
+            logger.warn(
+                'Failed to initialize Google Analytics handler, no tracking ID'
+            );
+
+            return;
+        }
+
+        this._enabled = true;
+        this._initGoogleAnalytics(options);
+    }
+
+    /**
+     * Initializes the ga object.
+     *
+     * @param {Object} options -
+     * @param {string} options.googleAnalyticsTrackingId - The GA track id
+     * required by the GA API.
+     * @returns {void}
+     */
+    _initGoogleAnalytics(options) {
+        /**
+         * TODO: Keep this local, there's no need to add it to window.
+         */
+        /* eslint-disable */
+        (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
```

Comment:
This can be moved somewhere outside the handler.

Reply:
Why? The handlers are suppose to provide specific implementations.